src/controls/ObjectControl.test.tsx

```diff
@@ -36,8 +36,8 @@ describe("ObjectControl", () => {
           uischema: objectUISchemaWithRule,
         })
 
-        expect(screen.getByText("Name")).not.toBeNull()
```

Comment:
This change fails tests for some reason. 🍿

Reply:
It probably needs to be a findBy. Will try that, but is this specific behavior something you disagree with or are you ok with the tests check passing as evidence this is a reasonable change?

Reply:
No, I just wanted to save this so I could once and for all figure out how this testing framework works.

Reply:
ah, I think it's actually not this change, but a change to cleanup, perhaps that is causing this issue